helix-term/src/ui/picker.rs

```diff
@@ -539,106 +557,90 @@ impl<T: Item + 'static> Picker<T> {
         // -- Render the contents:
         // subtract area of prompt from top
         let inner = inner.clip_top(2);
-
         let rows = inner.height;
         let offset = self.cursor - (self.cursor % std::cmp::max(1, rows as usize));
         let cursor = self.cursor.saturating_sub(offset);
+        let end = (offset + rows as usize).min(self.matcher.matches.len());
+        let mut indices = Vec::new();
+        let mut matcher_lock = MATCHER.lock();
+        let matcher = matcher_lock.get_or_insert_with(nucleo::Matcher::default);
+        // TODO: paths
+        matcher.config = MatcherConfig::DEFAULT;
+        if self.file_fn.is_some() {
+            matcher.config.set_match_paths()
+        }
```

Comment:
Is this TODO resolved (at least temporarily) by L569-571?

Reply:
Yeah forgot to remove that 😅